Vagrantfile

```diff
@@ -0,0 +1,46 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = 'holms/debian-jessie-amd64'
+  config.vm.box_url = "https://github.com/holms/vagrant-jessie-box/releases/download/Jessie-v0.1/Debian-jessie-amd64-netboot.box"
```

Comment:
To be honest this VM is on their ATLAS site.  I have a really hard time telling what is legit with Vagrant and what isn't.  If you have a recommendation I am all ears.  Could probably switch it over to https://atlas.hashicorp.com/ubuntu/boxes/trusty64 if you think that is better.  We can also host a box like that on s3 if needed to guarantee that it works.